Tests/fixtures/tv.yml

```diff
@@ -10064,3 +10064,47 @@
     model: ""
   os_family: Android
   browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (AppleTV11,1; CPU OS 11.1 like Mac OS X; en-US)
+  os:
+    name: tvOS
+    version: "11.1"
+    platform: ARM
+  client: null
+  device:
+    type: tv
+    brand: Apple
+    model: Apple TV 4K (2021)
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: RokuOS/3.0.0|Roku 3930X|STB|RokuOS 9.4.0 4200
+  os:
+    name: Roku OS
+    version: 3.0.0
```

Comment:
https://developer.roku.com/en-gb/docs/developer-program/release-notes/roku-os-release-notes.md

RokuOS 3. release in 2011
Roku Express (3930X)  3-gen release 2019
I think we should make a new regular expression for RokuOS

RokuOS/.+RokuOS (\d+\.[\d.]+)